static/templates/me_message.hbs

```diff
@@ -1,10 +1,16 @@
 <span class="message_sender no-select">
     <span class="sender_info_hover">
-        {{> message_avatar}}
+        <span title="{{t 'User profile' }} (u)">
+            {{> message_avatar}}
+        </span>
```

Comment:
Can't we just add the title attribute onto the `sender_info` to `message_sender` span here, rather than adding a new span?

Reply:
No, because that would add the title to the message as well (since this is a me message and the first line of the me message is a part of this) instead of just the sender name and avatar. We actually want to add the title only to the parts under `.sender-info-hover` but that cannot be done as explained in the commit message